src/sentry/snuba/entity_subscription.py

```diff
@@ -476,8 +477,46 @@ def _aggregate_query_results_v2(
         aggregated_results = [{col_name: crash_free_rate}]
         return aggregated_results
 
-    def get_snql_extra_conditions(self) -> List[Condition]:
-        return [Condition(Column("metric_id"), Op.EQ, resolve(self.org_id, self.metric_key.value))]
+    def build_query_builder(
```

Comment:
Should this be included in the PR?

Reply:
yep, i'm aware of it. Let me push current version.